client/consensus/manual-seal/Cargo.toml

```
@@ -22,19 +22,20 @@ parking_lot = "0.10.0"
 serde = { version = "1.0", features=["derive"] }
 assert_matches = "1.3.0"
 
-sc-client-api = { path = "../../../client/api" , version = "2.0.0-dev"}
-sc-transaction-pool = { path = "../../transaction-pool" , version = "2.0.0-dev"}
-sp-blockchain = { path = "../../../primitives/blockchain" , version = "2.0.0-dev"}
-sp-consensus = { package = "sp-consensus", path = "../../../primitives/consensus/common" , version = "0.8.0-dev"}
-sp-inherents = { path = "../../../primitives/inherents" , version = "2.0.0-dev"}
-sp-runtime = {  path = "../../../primitives/runtime" , version = "2.0.0-dev"}
-sp-core = {  path = "../../../primitives/core" , version = "2.0.0-dev"}
-sp-transaction-pool = { path = "../../../primitives/transaction-pool" , version = "2.0.0-dev"}
+sc-client-api = { path = "../../../client/api", version = "2.0.0-dev"}
```

A small grumble not introduced in this PR -- misses a space in the end.

```suggestion
sc-client-api = { path = "../../../client/api", version = "2.0.0-dev" }
```

And also below.